src/Core/AdminConsole/OrganizationFeatures/OrganizationUsers/DeleteManagedOrganizationUserAccountCommand.cs

```diff
@@ -50,31 +49,12 @@ public DeleteManagedOrganizationUserAccountCommand(
         _hasConfirmedOwnersExceptQuery = hasConfirmedOwnersExceptQuery;
         _referenceEventService = referenceEventService;
         _pushService = pushService;
-        _organizationRepository = organizationRepository;
         _providerUserRepository = providerUserRepository;
     }
 
     public async Task DeleteUserAsync(Guid organizationId, Guid organizationUserId, Guid? deletingUserId)
     {
-        var organizationUser = await _organizationUserRepository.GetByIdAsync(organizationUserId);
-        if (organizationUser == null || organizationUser.OrganizationId != organizationId)
-        {
-            throw new NotFoundException("Member not found.");
-        }
-
-        var managementStatus = await _getOrganizationUsersManagementStatusQuery.GetUsersOrganizationManagementStatusAsync(organizationId, new[] { organizationUserId });
-        var hasOtherConfirmedOwners = await _hasConfirmedOwnersExceptQuery.HasConfirmedOwnersExceptAsync(organizationId, new[] { organizationUserId }, includeProvider: true);
-
-        await ValidateDeleteUserAsync(organizationId, organizationUser, deletingUserId, managementStatus, hasOtherConfirmedOwners);
-
-        var user = await _userRepository.GetByIdAsync(organizationUser.UserId!.Value);
-        if (user == null)
-        {
-            throw new NotFoundException("Member not found.");
-        }
-
-        await _userService.DeleteAsync(user);
-        await _eventService.LogOrganizationUserEventAsync(organizationUser, EventType.OrganizationUser_Deleted);
+        await DeleteManyUsersAsync(organizationId, new[] { organizationUserId }, deletingUserId);
```

Comment:
I considered adding a test for this, but it's not straightforward since it calls another method within the same class. I explored using `Substitute.ForPartsOf<T>()`, but it requires making `DeleteManyUsersAsync` virtual. We could also mock everything in `DeleteManyUsersAsync`, but that seems more trouble than it's worth.